projects/plugins/protect/src/js/components/header/index.jsx

```diff
@@ -0,0 +1,69 @@
+import {
+	AdminSectionHero,
+	Status,
+	H3,
+	Container,
+	Col,
+	getIconBySlug,
+} from '@automattic/jetpack-components';
+import { useConnectionErrorNotice, ConnectionError } from '@automattic/jetpack-connection';
+import ScanNavigation from '../scan-navigation';
+import SeventyFiveLayout from '../seventy-five-layout';
+import styles from './styles.module.scss';
+
+const ConnectionErrorSection = () => {
```

Comment:
I had recently renamed this to `ScanSectionNotices` since it also contains the more general Jetpack admin notices container div. What do you think about extracting this to a separate `AdminSectionHeroNotices` component?

Reply:
Extracted in [d69c683](https://github.com/Automattic/jetpack/pull/39702/commits/d69c6831ee7d54bac3df1a37a130e91c9b6a91a1)